src/main/resources/application.yml

```diff
@@ -0,0 +1,160 @@
+server:
+  port: 9000
+  contextPath: 
+
+# use this as the security key for encryption
+security:
+  user:
+    password: 81d56928-891a-41b2-8dd2-132593e6d057
+
+spring:
+  main:
+    allow-bean-definition-overriding: true
+    allow-circular-references: true
```

Comment:
There needs to be an issue created for the `    allow-circular-references: true`.

Reply:
I went ahead and created the issue here: #178 .